ci/environment-release-build.yml

```diff
@@ -46,4 +45,5 @@ dependencies:
     - autoclasstoc
     - pydata_sphinx_theme
     - sphinxext-opengraph
+    - sphinx-panels
```

Comment:
sphinx-panels 0.6 does not appear to be available on conda-forge or defaults

Reply:
Yes, https://github.com/conda-forge/sphinx-panels-feedstock was last updated in January